ClearKeep/Scenes/TwoFactor/Views/TwoFactorContentView.swift

```diff
@@ -0,0 +1,215 @@
+			PasscodeTextField(text: $security, inputStyle: $securityStyle)
+			PasscodeTextField(text: $security, inputStyle: $securityStyle)
+		}
+		.frame(maxWidth: .infinity)
+	}
+
+	var titleView: some View {
+		HStack {
+			Text("2FA.Title".localized)
+				.font(AppTheme.shared.fontSet.font(style: .placeholder1))
+				.foregroundColor(foregroundMessage)
+			Spacer()
+		}
+	}
+
+	var resendCodeTitle: some View {
+		Text("2FA.DontGetCode".localized)
+			.font(AppTheme.shared.fontSet.font(style: .placeholder1))
+			.foregroundColor(foregroundMessage)
+	}
+
+	var buttonResend: some View {
+		Button {
+			print("Button tapped")
+		} label: {
+			Text("2FA.Resend".localized)
+				.padding(.all)
+				.font(AppTheme.shared.fontSet.font(style: .body2))
+				.frame(maxWidth: .infinity, alignment: .center)
+				.foregroundColor(foregroundColorMessage)
+		}
+	}
+}
+
+// MARK: - Private Func
+private extension TwoFactorContentView {
+	func customBack() {
+		self.presentationMode.wrappedValue.dismiss()
+	}
+}
+
+// MARK: - Support Variables
+private extension TwoFactorContentView {
+	var background: LinearGradient {
+		colorScheme == .light ? backgroundGradientPrimary : backgroundColorDark
+	}
+
+	var backgroundGradientPrimary: LinearGradient {
+		LinearGradient(gradient: Gradient(colors: AppTheme.shared.colorSet.gradientPrimary), startPoint: .leading, endPoint: .trailing)
+	}
+
+	var backgroundColorWhite: LinearGradient {
+		LinearGradient(gradient: Gradient(colors: [AppTheme.shared.colorSet.offWhite, AppTheme.shared.colorSet.offWhite]), startPoint: .leading, endPoint: .trailing)
+	}
+
+	var backgroundColorDark: LinearGradient {
+		LinearGradient(gradient: Gradient(colors: [AppTheme.shared.colorSet.black, AppTheme.shared.colorSet.black]), startPoint: .leading, endPoint: .trailing)
+	}
+
+	var backgroundColorGradient: LinearGradient {
+		LinearGradient(gradient: Gradient(colors: AppTheme.shared.colorSet.gradientPrimary), startPoint: .leading, endPoint: .trailing)
+	}
+
+	var backgroundColorView: LinearGradient {
+		colorScheme == .light ? backgroundColorWhite : backgroundColorGradient
+	}
+
+	var foregroundColorWhite: Color {
+		colorScheme == .light ? AppTheme.shared.colorSet.offWhite : AppTheme.shared.colorSet.offWhite
+	}
+
+	var foregroundColorPrimary: Color {
+		AppTheme.shared.colorSet.primaryDefault
+	}
+
+	var foregroundColorView: Color {
+		colorScheme == .light ? foregroundColorPrimary : foregroundColorWhite
+	}
+
+	var foregroundColorMessage: Color {
+		colorScheme == .light ? foregroundColorWhite : foregroundColorPrimary
+	}
+
+	var foregroundColorBackground: Color {
+		AppTheme.shared.colorSet.background
+	}
+
+	var foregroundColorGrey: Color {
+		AppTheme.shared.colorSet.grey1
+	}
+
+	var foregroundMessage: Color {
+		colorScheme == .light ? foregroundColorBackground : foregroundColorWhite
+	}
+}
+
+struct TwoFactorContentView_Previews: PreviewProvider {
+	static var previews: some View {
+		TwoFactorContentView()
+    }
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/c7185b19014dac546b06448508464c05a8c7eded/ClearKeep/Scenes/TwoFactor/Views/TwoFactorContentView.swift#L214"></span>Please use tabs instead of spaces<br />
<code>not_spaces</code> <code>TwoFactorContentView.swift:214</code></td>
    </tr>
  </tbody>
</table>